packages/core/src/Stores/client-store.js

```diff
@@ -2753,28 +2753,24 @@ export default class ClientStore extends BaseStore {
     setShouldShowEffortlessLoginModal(should_show_effortless_login_modal = true) {
         this.should_show_effortless_login_modal = should_show_effortless_login_modal;
     }
+
     async fetchShouldShowEffortlessLoginModal() {
-        if (this.is_passkey_supported) {
+        if (this.is_passkey_supported && this.root_store.ui.is_mobile && this.is_logged_in && this.is_authorize) {
```

Comment:
fetchShouldShowEffortlessLoginModal will be called only when this.is_logged_in && this.is_authorize are true
do we need to add these additional conditional here again?

Reply:
yeah, thought the same.. just left it here  to be more explicit.
but I think we can remove it 👍